### ISSUE TITLE
Nominate ellipse display

### ISSUE THREAD

**Issue by adamboche** [voteview/WebVoteView#259]
Do you need the oval? Could you use one vertical and one horizontal axis
to show medians or the zero points instead? Straighten out the Y and N
labels and and write out Yes/No.

**Comment by aaronrudkin** [comment]
I think this one is probably a non-starter. Because of how things are estimated, we are pretty wedded to the ellipse. I think changing the orientation of the Y/N labels would likely obscure that the line is dividing the two positions in the way it does -- I did try dropping the rotation, but then the placement was weird. I tried undoing the placement and putting them on the same X axis and that looks a little better, but I think less clear than the original mode (especially since depending on the line angle, one of the two labels is much closer to the line, which looks weird). Under these constraints, I tried changing Y/N to Yea/Nay and it looked much worse -- the "N" and "Y" scan properly at pretty much any angle but when you add the whole words your eyes get tricked by some of the angles.

If there's not a strong preference in favour of Alicia's suggestion here I think I will close without acting on this in a few days.
